backend/main.py

```
@@ -2,203 +2,381 @@
-    }
\ No newline at end of file
+@app.websocket("/ws/{drawing_id}")
+async def websocket_endpoint(websocket: WebSocket, drawing_id: str):
+    await manager.connect(websocket, drawing_id)
+    try:
+        while True:
+            # Ожидание данных от клиента (keep-alive или входящие команды)
+            await websocket.receive_text()
+    except WebSocketDisconnect:
+        manager.disconnect(websocket, drawing_id)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)